jetty-core/jetty-core-server/src/main/java/org/eclipse/jetty/core/server/handler/StatisticsHandler.java

```diff
@@ -103,6 +103,7 @@ public void failed(Throwable x)
     {
         private final LongAdder _bytesRead;
         private final LongAdder _bytesWritten;
+        private Callback _callback;
 
         public StatsRequest(Request request, LongAdder bytesRead, LongAdder bytesWritten)
```

Comment:
Class should be `private` and called `StatisticsRequest` similar to its enclosing `Handler`.